app/controllers/course/assessment/assessments_controller.rb

```diff
@@ -252,6 +315,32 @@ def valid_ordering?(proposed_ordering)
     question_assessments_hash.keys.sort == proposed_ordering.sort
   end
 
+  def create_koditsu_invitation_job
+    is_course_koditsu_enabled = current_course.component_enabled?(Course::KoditsuPlatformComponent)
+    is_assessment_koditsu_enabled = @assessment.koditsu_assessment_id && @assessment.is_koditsu_enabled
+
+    return unless is_course_koditsu_enabled && is_assessment_koditsu_enabled
+
+    return if Time.zone.now > @assessment.start_at - 12.hours
```

Comment:
This behaviour of creating invitations to Koditsu only if it's at least 12 hours before the `assessment.start_at` is not documented anywhere.  What's the rationale of only inviting 12 hours before?  
If I create the Koditsu PE <12 hours before the start, user has to manually sync? But user will not intuitively understand that they need to do this, or why they need to do this (sometimes).

Reply:
fixed.. if the current time is after 12 hrs before start of assessment, this function will call invite API directly

Reply:
Invites still being sent even after `assessment.end_at` if assessment is updated?